lldb/test/Shell/RPC/Generator/lit.local.cfg

```diff
@@ -0,0 +1,6 @@
+# All tests for the tool need lldb-rpc-gen to be built.
+# As far as I can see, you need to check all possible ways to
+# say "no" in a CMake option.
+negation_options = ["OFF", "NO", 0]
+if not config.lldb_has_lldbrpc or config.lldb_has_lldbrpc in negation_options:
```

Comment:
There is a list of variables in lldb/test/CMakeLists.txt that are passed to llvm_canonicalize_cmake_booleans. I think you need to add the new cmake option to that, then you'll not need to do this.